back/db/migrate/20251001090229_add_project_folder_id_index_to_nav_bar_items.rb

```diff
@@ -0,0 +1,7 @@
+class AddProjectFolderIdIndexToNavBarItems < ActiveRecord::Migration[7.1]
```

Comment:
The new strong migrations gem advised making this a separate migration: _"Adding an index non-concurrently blocks writes"_